mirascope/core/cohere/call_response.py

```diff
@@ -144,10 +143,20 @@ def tool(self) -> CohereTool | None:
 
     @classmethod
     def tool_message_params(
-        cls, tools_and_outputs: list[tuple[CohereTool, list[dict[str, Any]]]]
+        cls,
+        tools_and_outputs: list[
+            tuple[CohereTool, list[dict[str, Any]] | dict[str, Any] | str]
```

Comment:
I think we should limit the output type to just `str`. All of the other providers have just `str` except for Gemini which has `object`, but I think we should probably update that to just `str` as well so the interface matches.

For now, the answer to how to use the other supported types would be to construct your own provider specific tool results

Reply:
okay, done